### DISCUSSION TITLE
Python serverless functions

### DISCUSSION THREAD

**Discussion by proemian** [vercel/next.js#50263] [App Router]
- The app router documentation only describes `node.js` based endpoints, using [route handlers](https://nextjs.org/docs/app/building-your-application/routing/router-handlers) in `/app/api/...`
- Adding a python file under `/app/api/...` does not work
- Adding it to `/api/...` as described in the [serverless functions docs](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python) works, but then the other endpoints under `/app/api/...` no longer works

Is it possible to use python-based serverless functions together with the new route handlers, or do I need to stick with one or the other?

**Comment by superhorsy** [comment]
Same problem with "go" serverless function - looks lilie only one - /api or /app/api can be used at the same time. Let me know if solution is possible
